### ISSUE TITLE
Slideshow: allow to set duration images are shown

### ISSUE THREAD

**Issue by MarkWieczorek** [pulsejet/memories#813]
The slideshow feature shows each image for a duration that is hard coded into the software. It would be very helpful to allow the user to specify how long each image is shown. This could be specified in the "settings" field.

**Comment by jvlker** [comment]
Also options for repeat-all and random would be great! I would love to use the memories app as digital picture frame.

**Comment by larnoult** [comment]
I agree with @jvlker ; I'm interested in using the Memories app on an Android tablet to get a digital picture frame. How far is the app from such use case?

**Comment by funkyferdy** [comment]
I agree too to @jvlker!
Would be wonderfull to have the posibility to set slideshow duration and randomized output of an Album/Folder. Its the missing part for an digital picture frame (aka old Tablet or anything that can display a browser in Fullscreen)

**Comment by gbeliako** [comment]
By automatically repeating the photos and/or having random, it will be possible to make like a continuous slideshow to make a digital picture frame. And this could be further enhanced by nested albums.

**Comment by xfirf** [comment]
This would be really a cool feature. :-)

**Comment by robna** [comment]
plus one on this!
I also try to set up a digital picture frame with memories.
Is there anything planned?

I found two other issues ([photos app](https://help.nextcloud.com/t/gallery-slideshow-autoplay-speed-delay-setting/21078) and [viewer app](https://github.com/nextcloud/viewer/issues/447)) which address the same. There, some changes to the hard coded values have been proposed as a hacky workaround. However, the mentioned files `gallery/js/slideshowcontrols.js` or `viewer.js` don't appear to be present in a current NC installation.

For now I would be fine with changing the hardcoded values. Where would that need to be done for a slideshow presentation run in Memories app?

**Comment by shittybrakes** [comment]
Would be really cool to see this (repeating slideshows, timeline, live updates of slideshow) added at some point!

**Comment by petrtimr-crypto** [comment]
Also a PAUSE feature would be very appreciated as well!!! Spacebar for example etc.
